src/lib/ecore/ecore_time.c

```diff
@@ -55,14 +55,14 @@ ecore_time_get(void)
 ECORE_API double
 ecore_time_unix_get(void)
 {
-#ifdef HAVE_GETTIMEOFDAY
+//#ifdef HAVE_GETTIMEOFDAY
    struct timeval timev;
 
    gettimeofday(&timev, NULL);
    return (double)timev.tv_sec + (((double)timev.tv_usec) / 1000000);
-#else
-# error "Your platform isn't supported yet"
-#endif
+//#else
+//# error "Your platform isn't supported yet"
+//#endif
```

Comment:
Instead of commenting all this, you could make this be a warning instead of an error.